test/amperize.test.js

```diff
@@ -310,7 +384,7 @@ describe('Amperize', function () {
             .get('/images/IMG_xyz.jpg')
             .socketDelay(5500)
             .reply(200, {
-                data: '<Buffer 2c be a4 40 f7 87 73 1e 57 2c c1 e4 0d 79 03 95 42 f0 42 2e 41 95 27 c9 5c 35 a7 71 2c 09 5a 57 d3 04 1e 83 03 28 07 96 b0 c8 88 65 07 7a d1 d6 63 50>'
+                body: '<Buffer 2c be a4 40 f7 87 73 1e 57 2c c1 e4 0d 79 03 95 42 f0 42 2e 41 95 27 c9 5c 35 a7 71 2c 09 5a 57 d3 04 1e 83 03 28 07 96 b0 c8 88 65 07 7a d1 d6 63 50>'
```

Comment:
> can handle timeout errors

If i run all tests locally, this test does not wait 5s, it finishes immediately.

Reply:
It's supposed to do so, afaik. We only pretend that there's a timeout.

Reply:
That does not make sense to me. How do we ensure the timeout works? 🤔 
I guess you need to use `.delay` and increase the test timeout with `this.timeout`.

Reply:
Because the requested server returns a timeout error. This is what we mock.

Reply:
If we want to test, if `got` library handles timeout, then we can use `.delay` I think?

Reply:
Ok. Understood what you meant. I changed the test to simulate a real timeout with `.delay` and `this.timeout()`.